app/docs/src/reference/actions.md

```diff
@@ -0,0 +1,326 @@
+
+### Refresh Actions
+
+Refresh actions query your cloud provider to retrieve the current state of a resource and update the resource data in System Initiative. They ensure your model accurately reflects reality.
+
+Refresh actions can be triggered automatically after successful create or update actions or run manually.
+
+### Update Actions
+
+Update actions apply configuration changes to existing resources. They are automatically enqueued when you modify the properties of a component that already has an associated resource.
+
+Update actions only run after their dependencies succeed, ensuring that resources are updated in the correct order based on their relationships.
+
+### Delete Actions
+
+Delete actions are automatically enqueued when you delete a component that has an associated resource. They remove the actual infrastructure resource from your cloud provider.
+
+When you delete multiple related components, delete actions execute in reverse dependency order - for example, a subnet will be deleted before its VPC.
+
+### Manual Actions
+
+Manual actions are custom functions defined for specific components that don't fit into the standard create/update/delete lifecycle. For example, a manual action might restart a service, update a cluster configuration, or perform a one-time maintenance task.
+
+Manual actions must be explicitly enqueued to be run.
+
+## Action Lifecycle
+
+Actions follow a specific lifecycle within System Initiative:
+
+1. **Enqueuing**: Actions are automatically added to the change set when you create, modify, or delete components
+2. **Pending**: The action is queued and waiting to be executed
+3. **Execution**: When you apply the change set, actions run in dependency order
+4. **Completion**: Actions either succeed or fail
+5. **Persistence**: Failed actions remain enqueued until they succeed or are manually removed
+
+:::tip
+System Initiative records your intent to take an action, and it remains present on [HEAD](/reference/vocabulary#head) until it succeeds or is intentionally removed. This means you can retry failed actions without having to recreate them.
+:::
+
+## Action Execution Order
+
+Actions execute in an order determined automatically by the relationships between components. For example:
+
+- A VPC must be created before its subnets
+- Subnets must be created before instances that use them
+- Resources are deleted in reverse dependency order
+
+You don't need to manually specify action ordering - System Initiative determines the correct sequence based on subscriptions and configuration relationships between components.
+
+## Actions and Change Control
+
+Actions are only executed from HEAD - the change set representing "the real world." This ensures that all infrastructure changes flow through the change control process.
```

Comment:
I don't think this is right. HEAD doesn't need all actions passing nor all resources refresh. To me, HEAD is the read-only, finalized view of the workspace that contains your model's present state in your workspace. Everything in a change set is a simulation. HEAD is your model's present state, but not necessarily reality in that moment.